js/ui/gantt/ui.gantt.dialogs.js

```diff
@@ -120,12 +121,81 @@ class TaskEditDialogInfo extends DialogInfoBase {
                 min: 0,
                 max: 100
             }
+        }, {
+            dataField: "assigned.items",
+            editorType: "dxTagBox",
+            label: { text: "Resources" },
```

Comment:
Will this one (and other lines) be localized?

Reply:
While the widget is only available in English.